app/models/main.py

```diff
@@ -232,6 +237,16 @@ def validates_clarification_questions_open(self, key, value):
     def __repr__(self):
         return '<{}: {} slug={}>'.format(self.__class__.__name__, self.name, self.slug)
 
+    @validates('has_direct_award', 'has_further_competition')
+    def validates_framework_components(self, key, value):
+        has_direct_award = value if key == 'has_direct_award' else self.has_direct_award
+        has_further_competition = value if key == 'has_further_competition' else self.has_further_competition
+
+        if has_direct_award is False and has_further_competition is False:
+            raise ValidationError('At least one of `has_direct_award` and `has_further_competition` must be True.')
```

Comment:
These error messages talk about field names, but it seems from the tests that they make it through into potential API responses. In which case they are referring to the wrong field names - they would need to use the camelCased versions.

Not sure of the right solution here, maybe it has been done differently elsewhere tho?